tools/tty.js

```diff
@@ -6,7 +6,7 @@ try {
 } catch (e) {
     // ensure output buffers are flushed before process termination
     var exit = process.exit;
-    process.exit = function() {
+    if ("bufferSize" in process.stdout) process.exit = function() {
```

Comment:
@Jarred-Sumner this change is what concerns `bun` &minus; specifically to suppress workaround for Node.js TTY bug on OSX/macOS.